test/query_test.js

```diff
@@ -519,12 +519,17 @@ describe('query', function () {
     return Promise.all([p1, p2]);
   });
 
+  it('ref', function() {
+    var id = refN1.id.toString();
+    return assertQuery(query.Ref(classRef, query.Concat([id.charAt(0), id.substr(1)])), refN1);
```

Comment:
for some reason I thought if the ref was invalid it'd error out, but I guess that's not true